types/paypal__paypal-js/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for @paypal/paypal-js 1.0
+// Project: https://github.com/paypal/paypal-js
+// Definitions by: ReazerDev <https://github.com/ReazerDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Type definitions for npm package @paypal/paypal-js
+
+/// <reference types="node" />
```

Comment:
Where is this used?

Reply:
`// Type definitions for npm package @paypal/paypal-js` ?

Reply:
or `/// <reference types="node" />` ?

Reply:
GH includes the first three lines for context - the comment refers to the last line (in this case, `/// < reference`).

Reply:
Saw it in another definition lol Just took a look at some others and saw, that it's not there haha Gonna remove it